src/wp-admin/includes/file.php

```diff
@@ -1766,16 +1766,20 @@ function _unzip_file_pclzip( $file, $to, $needed_dirs = array() ) {
  *
  * @global WP_Filesystem_Base $wp_filesystem WordPress filesystem subclass.
  *
- * @param string   $from      Source directory.
- * @param string   $to        Destination directory.
- * @param string[] $skip_list An array of files/folders to skip copying.
+ * @param string $from      Source directory.
+ * @param string $to        Destination directory.
+ * @param array  $skip_list An array of files/folders to skip copying.
```

Comment:
Isn't this still an array of strings, i.e. `string[]` representing the path to the files/folders to skip?

Reply:
Isn't the param type supposed to identify the type of the param, not necessarily the contents of the param?

In other works, `$skip_list` is an array so this is correct. We don't specify the content type of the parameter.

Reply:
When an array is filled with the same data type, then we are identifying that data type.  In this case, it's an array of strings which is denoted as `string[]`.